app/styles/main.scss

```diff
@@ -323,6 +323,18 @@ form {
   }
 }
 
+reload div {
+  text-align: center;
+  padding: 10px 30px;
+  background-color: lighten($brand-primary-light, 15%);
```

Comment:
im assuming its just `$gray-light` and not `lighten($gray-light, 15%)` cause the latter is pretty much white lol. but i def prefer the yellow for sure 👍